src/command_modules/azure-cli-ams/azure/cli/command_modules/ams/_params.py

```diff
@@ -78,6 +79,15 @@ def load_arguments(self, _):  # pylint: disable=too-many-locals, too-many-statem
         c.argument('output_index', help='The element index of the output to remove.',
                    type=int, default=None)
 
+    with self.argument_context('ams transform output add') as c:
+        c.argument('preset', help='Allowed values: {}. In addition to the allowed values, you can also pass the local full path to a custom preset JSON file.'
```

Comment:
Just like when using enums, I'd rather you first describe the usage of the argument and end with Allowed values: {}. That's how all 'pseudo-enums' have been done too.